lib/ansible/plugins/connection/chroot.py

```diff
@@ -66,6 +66,7 @@ class Connection(ConnectionBase):
     become_methods = frozenset(C.BECOME_METHODS).difference(('su',))
 
     default_user = 'root'
+    chroot_program = 'chroot'
```

Comment:
```suggestion
    transport_cmd = 'chroot'
```
Tests were failing with traceback:
```
  File "/private/var/root/ansible/lib/ansible/executor/task_executor.py", line 139, in run
    res = self._execute()
  File "/private/var/root/ansible/lib/ansible/executor/task_executor.py", line 544, in _execute
    self._connection = self._get_connection(variables=variables, templar=templar)
  File "/private/var/root/ansible/lib/ansible/executor/task_executor.py", line 837, in _get_connection
    ansible_playbook_pid=to_text(os.getppid())
  File "/private/var/root/ansible/lib/ansible/plugins/loader.py", line 397, in get
    obj = obj(*args, **kwargs)
  File "/private/var/root/ansible/lib/ansible/plugins/connection/chroot.py", line 92, in __init__
    self._transport_cmd = distutils.spawn.find_executable(self.transport_cmd)
AttributeError: 'Connection' object has no attribute 'transport_cmd'
```